src/project.html

```diff
@@ -19,6 +19,97 @@
             </div>
         </div>
     </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12">
```

Comment:
All the projects should be wrapped in a single `col-12`, instead of each is wrapped in `col-12`. It's redundant.